bolt/org_config.go

```diff
@@ -0,0 +1,236 @@
+package bolt
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/boltdb/bolt"
+	"github.com/influxdata/chronograf"
+	"github.com/influxdata/chronograf/bolt/internal"
+)
+
+// Ensure OrganizationConfigStore implements chronograf.OrganizationConfigStore.
+var _ chronograf.OrganizationConfigStore = &OrganizationConfigStore{}
+
+// OrganizationConfigBucket is used to store chronograf organization configurations
+var OrganizationConfigBucket = []byte("OrganizationConfigV1")
+
+// OrganizationConfigStore uses bolt to store and retrieve organization configurations
+type OrganizationConfigStore struct {
+	client *Client
+}
+
+func (s *OrganizationConfigStore) Migrate(ctx context.Context) error {
+	return nil
+}
+
+// Get retrieves an OrganizationConfig from the store
+func (s *OrganizationConfigStore) Get(ctx context.Context, orgID string) (*chronograf.OrganizationConfig, error) {
+	var cfg chronograf.OrganizationConfig
+
+	err := s.client.db.View(func(tx *bolt.Tx) error {
+		return s.get(ctx, tx, orgID, &cfg)
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &cfg, nil
+}
+
+func (s *OrganizationConfigStore) get(ctx context.Context, tx *bolt.Tx, orgID string, cfg *chronograf.OrganizationConfig) error {
+	v := tx.Bucket(OrganizationConfigBucket).Get([]byte(orgID))
+	if len(v) == 0 {
+		return chronograf.ErrOrganizationConfigNotFound
+	}
+	return internal.UnmarshalOrganizationConfig(v, cfg)
+}
+
+// FindOrCreate gets an OrganizationConfig from the store or creates one if none exists for this organization
+func (s *OrganizationConfigStore) FindOrCreate(ctx context.Context, orgID string) (*chronograf.OrganizationConfig, error) {
+	var cfg chronograf.OrganizationConfig
+	err := s.client.db.Update(func(tx *bolt.Tx) error {
+		err := s.get(ctx, tx, orgID, &cfg)
+		if err == chronograf.ErrOrganizationConfigNotFound {
+			cfg = newOrganizationConfig(orgID)
+			return s.update(ctx, tx, &cfg)
+		}
+		return err
+	})
+
+	if err != nil {
+		return nil, err
+	}
+	return &cfg, nil
+}
+
+// Update replaces the OrganizationConfig in the store
+func (s *OrganizationConfigStore) Update(ctx context.Context, cfg *chronograf.OrganizationConfig) error {
```

Comment:
super nit picky, but `c` is probably sufficient as an argument name for the organization config.